advisories/a97090.md

```diff
@@ -12,7 +12,7 @@ Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
 
 ## Description
 
-In all versions of CockroachDB from [v19.1](../releases/v22.2.html#v19-1-0) to [v22.2.5](../releases/v22.2.html#v22-2-5), queries that used a zigzag [join](../{{site.versions["stable"]}}/joins.html) could return incorrect results in some cases. Zigzag joins can be planned when at least two columns from the same table are fixed to constant values by the query predicate, and indexes exist on both of those columns. The bug could occur when the two indexes used had a matching suffix of key columns where the direction of at least one column differed between the two indexes. 
+In all versions of CockroachDB from [v19.1](../releases/index.html#v19-1) to [v22.2.5](../releases/v22.2.html#v22-2-5), queries that used a zigzag [join](../{{site.versions["stable"]}}/joins.html) could return incorrect results in some cases. Zigzag joins can be planned when at least two columns from the same table are fixed to constant values by the query predicate, and indexes exist on both of those columns. The bug could occur when the two indexes used had a matching suffix of key columns where the direction of at least one column differed between the two indexes. 
```

Comment:
We'll be backporting a fix to 22.1 as well as 22.2, so 22.1.16 will include the fix. So I think you should update the "In all versions..." sentence to make that clear.